src/stream.cc

```diff
@@ -255,7 +255,8 @@ FileStream::~FileStream() {
 }
 
 void FileStream::Flush() {
-  if (file_) fflush(file_);
+  if (file_)
+    fflush(file_);
```

Comment:
Again, I thought the local style was to require braces around blocks.

Reply:
As explained in https://github.com/WebAssembly/wabt/pull/1684#discussion_r660172600, it is clang-tidy's job. For this PR, I allowed one-line short ifs to match the codebase.